### ISSUE TITLE
About Inference code

### ISSUE THREAD

**Issue by yiluzhou1** [dek924/PerX2CT#5]
Dear authors, 

Thank you for your fantastic work! Could you please share the inference code? If using the Pretrained Model (suppose using your uploaded one), how to use only two X-rays (PA and lateral view images) to reconstruct a 3D CT volume? Many thanks!

**Comment by dek924** [comment]
Our inference code consists of `main_test.py` and `main_test_zoom.py`. To evaluate our pre-trained checkpoint, please refer to the [Evaluation](https://github.com/dek924/PerX2CT#full-frame-evaluation-default:~:text=PerX2CT_global_w_zoomin.yaml.-,Evaluation,-Full%20frame%20evaluation) section in the README, where we have updated the description. Additionally, if you want to generate a single CT volume from a single pair of X-rays (PA, AP), please try to keep only one target CT slice in `dataset_list/LIDC_CTslice/test_sorted.txt` or `val_sorted.txt`. Thank you.

**Comment by yiluzhou1** [comment]
Dear author,
Thank you so much for your response! I've tested using the following command:
`python main_test.py --ckpt_path "checkpoints/PerX2CT.ckpt" --config_path "configs/PerX2CT.yaml" --save_dir "temp/" --val_test "val"`
I put the following (PA, lateral) in the text file you mentioned:
```
./data_preprocessing/lidc_idri/LIDC-HDF5-256_ct320_ct128_CTSlice/LIDC-IDRI-0001.20000101.3000566.1/ct/coronal_064.h5
./data_preprocessing/lidc_idri/LIDC-HDF5-256_ct320_ct128_CTSlice/LIDC-IDRI-0001.20000101.3000566.1/ct/sagittal_064.h5
```
"val_OutputFull_GTFull.png" is generated, but it seems there are only 2 slices. How to generate more slices to make a 3D CT volume?
![val_OutputFull_GTFull](https://github.com/dek924/PerX2CT/assets/135764391/3aa12718-462f-4c51-861f-3f411671477f)

I checked your "main_test.py". The "run_test_ramdom_image" function is responsible for generating images. It looks like it can generate only 2 slices? As the shape of "test_output_img_full" is (2, 128, 128). 
![image](https://github.com/dek924/PerX2CT/assets/135764391/55dd9d8f-6fb3-45c8-9aac-8f13ed544f96)

Another question: Instead of using 2 slices from CT, I want to use 2 orthogonal DRR projected Xray images (PA, lateral) to generate 3D CT volume, I should convert the DRR image to H5 format first, correct?
Thank you!
